_episodes/09-conflict.md

```diff
@@ -158,8 +158,12 @@ Automatic merge failed; fix conflicts and then commit the result.
 ~~~
 {: .output}
 
-`git pull` tells us there's a conflict,
-and marks that conflict in the affected file:
+The `git pull` command combines a `git fetch` with a `git merge` to update the local repository to include
```

Comment:
I'm not sure whether introducing `git fetch` and `git merge` is good. People may get confused and think they miss the part where the commands where explained. What do you think about talking about "fetching" and "merging" without mentioning them as `git` commands? 

A few lines later with talk about fetching the repo and a few lines before we talk about mergin changes.

Reply:
I agree with this. I think this will be clear without the specific reference regarding git pull as a git fetch + git merge.

Reply:
@JamesEMcClure, great! Could you make the changes in your PR? I think "git pull" to "pull" and "git merge" to "merge" will make it.

Reply:
I've updated my pull request so that the specific references to the "git fetch" and "git merge" commands no longer appear.